src/test/java/bio/terra/service/resourcemanagement/BucketResourceLockTester.java

```diff
@@ -47,7 +47,7 @@ public void run() {
     try {
       // create the bucket and metadata
       bucketResource =
-          bucketService.getOrCreateBucket(bucketName, projectResource, region, flightId);
+          bucketService.getOrCreateBucket(bucketName, projectResource, region, flightId, null);
```

Comment:
I think this is going to create a bunch of entries in the DB that will never get cleaned up...is that accurate?

Reply:
Doesn't matter. The database goes away anyways in tests now!

Reply:
I'm not worried about the test DB, I'm worried about application db

Reply:
Why would this cause a bunch of entries?

Reply:
I think I was worried about the case where we create a bucket entry for each export but if there's just one per snapshot it's probably ok.  And we have to link that bucket with with the snapshot somehow (and make sure that it gets deleted when we delete a snapshot)

Reply:
It'll get deleted when the project is deleted.

Reply:
From the database?

Reply:
The bucket is in the same project as the snapshot, and since we now have 1 project per snapshot, we should probably be deleting the entire google project on snapshot delete.